cmd/lenses-cli/configuration.go

```diff
@@ -99,6 +99,14 @@ func newConfigurationManager(set *pflag.FlagSet) *configurationManager {
 	return m
 }
 
+func newEmptyConfigManager() *configurationManager {
```

Comment:
I see it is used only in one place, so may a method for this is redundant. As I understand your intent was to make it re-usable right?

Reply:
Removing it.